packages/eas-cli/src/graphql/queries/AppQuery.ts

```diff
@@ -60,4 +73,64 @@ export const AppQuery = {
     assert(data.app, 'GraphQL: `app` not defined in server response');
     return data.app.byFullName;
   },
+  async byIdWorkflowsAsync(
+    graphqlClient: ExpoGraphqlClient,
+    appId: string
+  ): Promise<AppWorkflowsFragment> {
```

Comment:
nit: update to return array since method name indicates that it does
```suggestion
  ): Promise<AppWorkflowsFragment['workflows']> {
```